src/core/reference/src/runtime/reference/utils/fft_common.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/runtime/reference/utils/fft_common.hpp"
+
+#include <algorithm>
+#include <cassert>
+#include <cstddef>
+#include <cstring>
+#include <numeric>
+
+#include "ngraph/check.hpp"
+
+namespace ngraph {
+namespace runtime {
+namespace reference {
+namespace fft_common {
+std::vector<int64_t> reverse_shape(const ngraph::Shape& shape) {
+    std::vector<int64_t> reversed_shape(shape.begin(), shape.end() - 1);
+    std::reverse(reversed_shape.begin(), reversed_shape.end());
+    return reversed_shape;
+}
+
```

Comment:
```suggestion
```

Reply:
This method is common with the nGraph reference implementation of (I)DFT [see `openvino/src/core/reference/src/runtime/reference/fft.cpp`]. All such methods were removed from `openvino/src/core/reference/src/runtime/reference/fft.cpp`. The method `reverse_shape` will be renamed.